### DISCUSSION TITLE
Cannot Connect to Netlify

### DISCUSSION THREAD

**Discussion by kennyheimbuch** [GetPublii/Publii#1599] [Help]
Moved my Publii install to a new computer, site backup restore worked perfectly. Problem now is that the site cannot connect to Netlify, even though the domain, Site ID and access token are entered correctly (no trailing spaces etc)

Even generating new access token does nothing.

When trying to "test connection" it does this forever with no error. Saving settings and trying to sync site generates error.

Any ideas what is causing this?

Also, in the section in Netlify where the access tokens are created - how does that platform "know" which token is connected to which website/app etc?

also, there is an error in the deployment-process log:

Request failed (URL: https://api.netlify.com/api/v1/sites//deploys): Fetch HTTP Error: 404
[Mon, 05 Aug 2024 13:50:50 GMT] Netlify ERROR: TypeError: Cannot read properties of undefined (reading 'id')

**Comment by Monkeyboy307** [comment]
I have the same problem. Token and site ID are both correct. The publii app hangs when testing connection. It never seems to connect. Unlike you, I haven't switched anything to a new computer, so we can probably eliminate that as causing the problem.

I currently have to export my site to a zip and upload it direct to Netlify.

**Reply by kennyheimbuch** [reply]
Eventually, despite the app hanging, when I went back and synched the site, it worked. Maybe some part of the authentication test changed.

**Reply by Monkeyboy307** [reply]
You're right! Just tried it myself and the website synched fine, despite "test connection" not working properly.

**Reply by kennyheimbuch** [reply]
Good news!  At least the upload works. Maybe it is something on Netlify's end that is causing the endless  authentication loop.
